src/Helpers/__tests__/Hooks.test.js

```diff
@@ -0,0 +1,31 @@
+import { useRbac } from '../Hooks';
+import { PERMISSIONS } from '../constants';
+import { renderHook } from '@testing-library/react-hooks';
+
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  __esModule: true,
+  default: () => ({
+    getUserPermissions: () => Promise.resolve([{ permission: 'inventory:hosts:read' }, { permission: 'blob:*' }, { permission: 'test:read' }]),
+  }),
+}));
+
+describe('useRbac', () => {
+  it('should return correct values', async () => {
+    const { result, waitForNextUpdate } = renderHook(() => useRbac([PERMISSIONS.readHosts], 'inventory'));
+    expect(result.current).toEqual([[[]], true]);
+    await waitForNextUpdate();
+    expect(result.current).toEqual([[true], false]);
+  });
+
+  it('should work with * permission', async () => {
```

Comment:
what the asterisk means? for all permissions?

Reply:
Yep, I will change this description to be more understandable.